### ISSUE TITLE
Show the current location & total length of videos

### ISSUE THREAD

**Issue by mkolodny** [feross/webtorrent#727]
* WebTorrent version: 0.3.3
* Node.js version: 5.8.0
* Browser name/version (if using WebTorrent in the browser): Desktop (Mac 10.11.4)

First off, amazing work!

I have a two part feature request:

1. It would be great to know where you're at in the video you're watching. That way if you need to stop watching during the video for some reason, or your wifi cuts out, you can easily get back to where you left off.

2. I'd love to know the total length of the video. It's super helpful for planning (i.e. deciding whether you have enough time to watch a video, and whether a video is worth watching). This could be "--:--" until the video finishes downloading.

So basically as close to the way YouTube shows <current location>/<total length> as we can get.

**Comment by rom1504** [comment]
If I understand correctly this should go in https://github.com/feross/webtorrent-desktop/issues ;)

**Comment by feross** [comment]
There's an issue on the `webtorrent-desktop` repo to track this feature request: https://github.com/feross/webtorrent-desktop/issues/351
